packages/mobile/src/screens/explore-screen/tabs/PlaylistsTab.tsx

```diff
@@ -32,12 +33,14 @@ export const PlaylistsTab = () => {
   }, [exploreStatus, dispatch])
 
   return (
-    <CollectionList
-      isLoading={
-        exploreStatus === Status.LOADING || playlistsStatus !== Status.SUCCESS
-      }
-      ListHeaderComponent={<TabInfo header={messages.infoHeader} />}
-      collection={playlists}
-    />
+    <ScrollView>
+      <TabInfo header={messages.infoHeader} />
```

Comment:
I feel pretty strongly this should be in the flat list header, what's the main reason this doesn't work?

Reply:
oh thats weird. I updated it, but it reverted somehow. I'll fix 👍